verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -124,6 +128,9 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
             if (isUndefined(oldRowData)) {
                 return reject();
             }
+            if (!this.validateRequired(newRowData.source_url)) {
+                return reject();
```

Comment:
Hmm a lot of existing data doesn't have a source, that will make us unable to add it if we wanted to.

@khmoran should source be required?

Reply:
Ah true. I added the validator because the API call failed without a source. I think it makes sense that all fields that are manually added from the UI should require a source.

Reply:
My thinking was that all new data should require a source -- but not existing data. So the mongodb validator schema doesn't require it, but Mongoose should on insert, IMO.

Reply:
But also if any of the current validation is impeding development, happy to ice it for now and bring it back when things are more stable.